core/embed/rust/src/ui/model_mercury/layout.rs

```diff
@@ -2082,9 +2081,9 @@ pub static mp_module_trezorui2: Module = obj_module! {
     /// def show_share_words(
     ///     *,
     ///     title: str,
-    ///     pages: Iterable[str],
+    ///     share_words: Iterable[str],
```

Comment:
Breaks style check: https://github.com/trezor/trezor-firmware/actions/runs/8491733359/job/23263990332?pr=3656

Probably best to name the function something new, until we have modtrezorui-per-model.

Reply:
Fixed here. Also the device tests for reset are running. I haven't updated fixtures though.
5b50beaa5de417834f97062b346aa1926f2fe120